tutorials/asr/Multilang_ASR.ipynb

```diff
@@ -1713,7 +1713,12 @@
    },
    "outputs": [],
    "source": [
-    "asr_model.setup_multiple_validation_data(val_data_config=validation_ds) "
+    "asr_model.setup_multiple_validation_data(val_data_config=validation_ds) \n",
```

Comment:
Is it possible to automate this and hide this stuff?

Reply:
That would be difficult because originally the notebook is setting up multiple validation dataloaders externally after init 
```
asr_model.setup_multiple_validation_data(val_data_config=validation_ds) 
```
In a normal case this would be taken care of during init itself -  https://github.com/NVIDIA/NeMo/blob/main/nemo/core/classes/modelPT.py#L185

Reply:
Ideally IMHO we should be doing the validation_step_outputs checking and update inside the setup_multiple_validation_data() and setup_multiple_test_data() functions and not outside in init. But for that we'll have to refactor the code a bit and fix other functions/notebooks which might be affected by this change

Reply:
@titu1994 yeah as Kunal said, its better to move setting of `validation_step_outputs` to validation setup and `test_step_outputs` to test setup. @athitten

Reply:
No objection!

Reply:
Seriously, let's fix this inside multi validation and test function.  Even temporarily having this gnarly code used visible is bad

Reply:
Adding @athitten to the discussion as she originally worked on the PTL2 refactor which would get affected if we move setting of `validation_step_outputs` to setup_multiple_validation_data() and `test_step_outputs` to setup_multiple_test_data() in core modelPT [https://github.com/NVIDIA/NeMo/blob/main/nemo/core/classes/modelPT.py](https://github.com/NVIDIA/NeMo/blob/main/nemo/core/classes/modelPT.py).